src/NUnitFramework/framework/Attributes/ExplicitAttribute.cs

```diff
@@ -62,8 +62,8 @@ public ExplicitAttribute(string reason)
         /// </summary>
         /// <param name="test">The test to modify</param>
         public void ApplyToTest(Test test)
-        {
-            if (test.RunState != RunState.NotRunnable)
+        {
+            if (test.RunState != RunState.NotRunnable && test.RunState != RunState.Ignored)
```

Comment:
As I understand from https://github.com/nunit/nunit/issues/48 it's right behaviour.
![image](https://cloud.githubusercontent.com/assets/1849690/7240631/2e52060e-e7bb-11e4-8352-429884a05c6c.png)